miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppScheme.kt

```diff
@@ -48,4 +47,15 @@ internal class MiniAppScheme private constructor(miniAppId: String) {
         it.data = url.toUri()
         context.startActivity(it)
     }
+
+    internal fun openMaiComposer(context: Context, url: String) = Intent(Intent.ACTION_SEND).let {
+        val mail = MailTo.parse(url)
+        it.putExtra(Intent.EXTRA_EMAIL, arrayOf(mail.to))
+        it.putExtra(Intent.EXTRA_TEXT, mail.body)
+        it.putExtra(Intent.EXTRA_SUBJECT, mail.subject)
+        it.putExtra(Intent.EXTRA_CC, arrayOf(mail.cc))
+        it.putExtra(Intent.EXTRA_BCC, arrayOf(mail.headers["bcc"]))
+        it.type = "message/rfc822"
+        context.startActivity(it)
```

Comment:
Could we check if the `context` is okay before starting it with the intent e.g.

```
if (it.resolveActivity(context.packageManager) != null) {
   context.startActivity(it)
}
```

Reply:
Good idea.